authentication/README.md

```diff
@@ -21,6 +21,18 @@ Getting Started
 - Run the app on your device or simulator.
     - Select **Sign In** and select Google to begin.
 
+### Microsoft Sign In Setup
+- Go to the [Firebase Console](https://console.firebase.google.com) and navigate to your project:
+  - Select the **Auth** panel and then click the **Sign In Method** tab.
+  - Open panel on bottom right corner and click **Experiment Panel**.
+  - Enable **FirebaseAuthProvidersExperimentEnabled** experiment
```

Comment:
Are these going to be the public instructions?

Reply:
Just double checked with our TL and you are right, this is not needed when we launch it. I modified the instruction.